docs/advanced/pdf.md

```diff
@@ -7,15 +7,22 @@ page describes a few ways to do so.
 PDF building is experimental, and may change or have bugs.
 ```
 
+There are two approaches to building PDF files:
+
+```{contents}
+depth: 1
+```
```

Comment:
these don't seem to be making it into the built HTML (see the netlify preview)

Reply:
@choldgraf this is a sphinx directive. I wonder why it isn't working. Might take a while to diagnose shall I remove or keep in for now and add a comment?